charts/incubator/pastey/questions.yaml

```diff
@@ -10,64 +10,117 @@ questions:
 # Include{replicas1}
 # Include{podSpec}
 # Include{containerMain}
+
+                                - variable: imageSelector
+                                  label: Select Image
+                                  schema:
+                                    type: string
+                                    default: image
+                                    enum:
+                                      - value: image
+                                        description: Image
+                                      - value: imageSim
+                                        description: Slim
+
+                                - variable: env
+                                  label: Image Environment
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
+                                      - variable: PASTEY_LISTEN_ADDRESS
+                                        label: "Pastey Listen Address"
+                                        description: "Address to listen on."
+                                        schema:
+                                          type: string
+                                          $ref:
+                                            - definitions/nodeIP
```

Comment:
If thats the address for inside the container, and someone sets it to the node IP it will never work

Reply:
should it be removed then? defaults to 0.0.0.0